editor/plugins/version_control_editor_plugin.cpp

```diff
@@ -120,7 +112,8 @@ void VersionControlEditorPlugin::_initialize_vcs() {
 		return;
 	}
 
-	int id = set_up_choice->get_selected_id();
+	// There's no placeholder option, so add 1 to the real value
+	const int id = set_up_choice->get_selected_id() + 1;
```

Comment:
Replacing with `const int id = set_up_choice->get_selected_id();` seems to get rid of the crash and select the addon properly